AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_DisplayMapperAdded.py

```diff
@@ -129,52 +188,233 @@ def AtomEditorComponents_DisplayMapper_AddedToEntity():
+                AtomComponentProperties.display_mapper('Min Point (luminance)'), value=4.8)
+            Report.info(display_mapper_component.get_component_property_value(
+                AtomComponentProperties.display_mapper('Min Point (luminance)')))
+            Report.result(Tests.luminance_level_min, Math_IsClose(display_mapper_component.get_component_property_value(
+                AtomComponentProperties.display_mapper('Min Point (luminance)')), 4.8, TOLERANCE))
+            display_mapper_component.set_component_property_value(
+                AtomComponentProperties.display_mapper('Min Point (luminance)'), value=0.002)
+            Report.result(Tests.luminance_level_min, Math_IsClose(display_mapper_component.get_component_property_value(
+                AtomComponentProperties.display_mapper('Min Point (luminance)')), 0.002, TOLERANCE))
+
+            # 15. 'Mid Point (luminance)' set high and low value then set default
+            display_mapper_component.set_component_property_value(
+                AtomComponentProperties.display_mapper('Mid Point (luminance)'), value=1005.0)
+            Report.result(Tests.luminance_level_mid, Math_IsClose(display_mapper_component.get_component_property_value(
+                AtomComponentProperties.display_mapper('Mid Point (luminance)')), 1005.0, TOLERANCE))
+            display_mapper_component.set_component_property_value(
+                AtomComponentProperties.display_mapper('Mid Point (luminance)'), value=0.002)
+            Report.result(Tests.luminance_level_mid, Math_IsClose(display_mapper_component.get_component_property_value(
+                AtomComponentProperties.display_mapper('Mid Point (luminance)')), 0.002, TOLERANCE))
+            display_mapper_component.set_component_property_value(
+                AtomComponentProperties.display_mapper('Mid Point (luminance)'), value=4.8)
+
+            # 16. 'Max Point (luminance)' set high and low value then set default
+            display_mapper_component.set_component_property_value(
+                AtomComponentProperties.display_mapper('Max Point (luminance)'), value=4000.0)
+            Report.result(Tests.luminance_level_max, Math_IsClose(display_mapper_component.get_component_property_value(
+                AtomComponentProperties.display_mapper('Max Point (luminance)')), 4000.0, TOLERANCE))
+            display_mapper_component.set_component_property_value(
+                AtomComponentProperties.display_mapper('Max Point (luminance)'), value=0.002)
+            Report.result(Tests.luminance_level_max, Math_IsClose(display_mapper_component.get_component_property_value(
+                AtomComponentProperties.display_mapper('Max Point (luminance)')), 0.002, TOLERANCE))
+            display_mapper_component.set_component_property_value(
+                AtomComponentProperties.display_mapper('Max Point (luminance)'), value=1005.7191162)
+
+            # 17. 'Surround Gamma' set high and low value
```

Comment:
or you could use this approach for the "True/False" steps instead and just have the main `# 17. 'Surround Gamma' set high and low value` then all the code for that step underneath it

Reply:
all comments updated for consistency